apps/state_mediator/test/state_mediator/mqtt_mediator_test.exs

```diff
@@ -55,8 +55,15 @@ defmodule StateMediator.MqttMediatorTest do
 
     test "are sent to the state module" do
       {:ok, _pid} = start_link(@opts)
-      assert_receive {:updated, contents}, 5_000
-      assert <<_::binary>> = contents
```

Comment:
suggestion: you've dropped the assertions which are actually about the state module (these ones above).